test/sanitytest.c

```diff
@@ -49,6 +49,15 @@ static int test_sanity_enum_size(void)
     return 1;
 }
 
+static int test_sanity_int_size(void)
+{
+    if (!TEST_size_t_eq(sizeof(int), 4)
+        || !TEST_int_eq(CHAR_BIT, 8)
+        || !TEST_size_t_lt(sizeof(int), sizeof(int64_t)))
```

Comment:
Why did you keep `!TEST_size_t_eq(sizeof(int), 4)`?  This line is meant to replace it, and it really makes no sense at all to keep both...